packages/squiggle-lang/src/reducer/index.ts

```diff
@@ -59,6 +61,7 @@ function throwFrom(
  * `context.evaluate` can inject additional behaviors, e.g. delay for pseudo-async evaluation.
  */
 export const evaluate: ReducerFn = (expression, context) => {
+  jstat.setRandom(context.rng);
```

Comment:
`jstat` is global, so this is stateful, but as long as `evaluate` is not async, it's good enough.

Reply:
Makes sense. A bit scary, we'll just have to remember if we go async or parallelize